README.md

```diff
@@ -19,28 +19,28 @@ Packagist Stats was created by, and is maintained by [Graham Campbell](https://g
 
 This project is relatively small, and it's usage is pretty simple.
 
-Install dependencies:
+Installation via composer:
 
 ```bash
-$ composer install --no-dev
+$ composer global require graham-campbell/packagist-stats dev-master
```

Comment:
Composer will resolve a version to use. Even then, you should NEVER need dev-master.

`^0.1.*@alpha` or `^0.2.*@dev` will both work fine.